utils/formatResultsData.js

```
@@ -27,15 +29,33 @@ function formatResultData({ res, total = null, limitNum = null, pageNum = null,
     active: false
   });
 
+  const formatResult = Array.isArray(result) ? result.map((item) => {
+    try {
+      if (item.hasOwnProperty('money')) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)